QUEST_LV_0100.tsv

```diff
@@ -6403,101 +6403,101 @@ QUEST_LV_0100_20150317_006402	{memo X}Mr. Juan asked you to bring him Yukopus Le
 QUEST_LV_0100_20150317_006403	{memo X}Acquired Yukopus Leather! Bring it to Mr. Juan.
 QUEST_LV_0100_20150317_006404	{memo X}Mr. Juan asked you to bring him Vubbe Theif's Leather to make the armor.
 QUEST_LV_0100_20150317_006405	{memo X}Acquired Vubbe Thief's Leather! Bring it to Mr. Juan.
-QUEST_LV_0100_20150323_006406	Now take the unsealing stone and go to Ramstis Ridge. {nl}Student of Gustas will teach you how to release the seal there. 
-QUEST_LV_0100_20150323_006407	Were you not guided yet?{nl}It's the student of Gustas in Ramstis Ridge.{nl}It is made to let only the chosen one enter.
-QUEST_LV_0100_20150323_006408	Ask the mayor of the Miners' Village for directions to Gele Plateau. {nl}I will sent the message to the Paladin Master first. {nl}May the blessings of Goddess be with you.
+QUEST_LV_0100_20150323_006406	$Now take the unsealing stone and go to Ramstis Ridge. {nl}Student of Gustas will teach you how to release the seal there. 
```

Comment:
"Student of Gustas will teach you how to release the seal there." -> "The student of Gustas will teach you how to release the seal there."